libdrgn/lazy_parameter.c

```diff
@@ -45,3 +74,21 @@ struct drgn_error *drgn_lazy_parameter_get_type(struct drgn_lazy_parameter *lazy
 	return NULL;
 }
 
+struct drgn_error *drgn_lazy_parameter_get_object(struct drgn_lazy_parameter *lazy_parameter,
```

Comment:
This should probably return an error if the parameter is not an object. Right now it'll just crash.